package.json

```diff
@@ -6,7 +6,8 @@
     "repository": "git@github.com:BolajiOlajide/ng-faker.git",
     "contributors": [
         "Bolaji Olajide",
-        "Rotimi Babalola"
+        "Rotimi Babalola",
+        "Oyindamola Subair"
```

Comment:
I am yet to contribute

Reply:
But you will right? @oyinda-subair 

Adding here so won't have to bother about it later.
Cool?